### PR TITLE
Rename actions unit to `repo.actions` and add docs for it

### PR DESCRIPTION
**Pull request by wolfogre**
I neglected that the `NameKey` of `Unit` is not only for translation, but also configuration. So it should be `repo.actions` to maintain consistency.

## :warning: BREAKING :warning:

If users already use `actions.actions` in `DISABLED_REPO_UNITS` or `DEFAULT_REPO_UNITS`, it will be treated as an invalid unit key.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: models/unit/unit.go [hunk 1]
```diff
@@ -312,7 +312,7 @@ var (
 
 	UnitActions = Unit{
 		TypeActions,
-		"actions.actions",
```
Comment (delvh) [inline models/unit/unit.go:315]: Can we delete this source string now?

**Review by delvh — APPROVED** [review]
The good thing is:
Since it was previously undocumented, there likely aren't a lot of affected users.
I mean, I myself didn't know that this was possible, and I reviewed the original PR and have gotten somewhat familiar with the code base :sweat_smile:

**Comment by yardenshoham** [comment]
Backporting a breaking PR?

File: models/unit/unit.go (hunk 1 above)
Comment (wolfogre) [reply models/unit/unit.go:315]: It has been used elsewhere.

**Comment by wxiaoguang** [comment]
We have the `backport-locales.go`, so I guess backporting some translations are feasible now.

**Review by techknowlogick — DISMISSED** [review]
Blocking, as we shouldn't be backporting breaking changes. Is there a way to make the previous option an alias for the update option?

**Comment by lunny** [comment]
> Blocking, as we shouldn't be backporting breaking changes. Is there a way to make the previous option an alias for the update option?

Since we have said actions is an experiment feature in v1.19, I think we could change it directly.

BTW: maybe we should also update `repo_unit` table? @wolfogre

**Comment by yp05327** [comment]
Just a question. Is it necessary/possible to change `action_XXX` to `repo_action_XXX` in db too, as we also have `activities.action` which has `action` table in db.

**Comment by lunny** [comment]
> 

It should be another PR. Maybe we can rename the old table `action` to another name.

**Comment by wolfogre** [comment]
> BTW: maybe we should also update `repo_unit` table?

@lunny 

Update what?

```go
type RepoUnit struct { //revive:disable-line:exported
	ID          int64
	RepoID      int64              `xorm:"INDEX(s)"`
	Type        unit.Type          `xorm:"INDEX(s)"`
	Config      convert.Conversion `xorm:"TEXT"`
	CreatedUnix timeutil.TimeStamp `xorm:"INDEX CREATED"`
}
```

It doesn't use the key string, just `unit.Type`. And if an existing repo has Actions enabled, we shouldn't touch it.

**Comment by wxiaoguang** [comment]
By the way, there is another strong reason about why this "breaking" could be backported: Gitea declares that the Actions feature is experimental, so it's not surprised that the stable release needs some "breaking" fixes for such experimental feature.

**Comment by lunny** [comment]
@techknowlogick please review again. I think the break is just an option on the ini file from `actions.actions` to `repo.actions`. @wolfogre

**Comment by 6543** [comment]
also ok with backpor that ...

**Pull request merged**